browser/brave_search/BUILD.gn

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2019 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+source_set("brave_search") {
+  sources = [
+    "backup_results_navigation_throttle.cc",
+    "backup_results_navigation_throttle.h",
+    "backup_results_service_factory.cc",
+    "backup_results_service_factory.h",
+    "backup_results_service_impl.cc",
+    "backup_results_service_impl.h",
+  ]
+
+  deps = [
+    "//brave/components/brave_search/browser",
+    "//brave/components/brave_search/common",
+    "//brave/components/brave_shields/content/browser",
+    "//chrome/browser/content_extraction",
+    "//chrome/browser/content_settings:content_settings_factory",
+    "//chrome/browser/profiles",
+    "//components/keyed_service/core",
+    "//content/public/browser",
+    "//services/network/public/cpp",
+  ]
+}
+
+if (!is_android) {
```

Comment:
why does this test not run on android? In a follow-up please convert it to run on android. Also fyi using `!is_android` around browser tests is not an alternative to adding tests to android_browser_tests.gni. There is code owner for that file specifically to enforce that new tests run on android where applicable.